go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -1294,27 +1294,55 @@ func (tsv *TabletServer) convertAndLogError(ctx context.Context, sql string, bin
 	if err == nil {
 		return nil
 	}
-	err = tsv.convertError(ctx, sql, bindVariables, err)
 
-	if logStats != nil {
-		logStats.Error = err
-	}
-	errCode := vterrors.Code(err)
+	errStr := err.Error()
```

Comment:
This can be moved below to the line before it's first used. I actually don't think you need this variable any more. Maybe just use err.Error() in the format call?

Reply:
Good idea -- updated.